### PR TITLE
React router: v6.9.0 is automated

### PR DESCRIPTION
**Pull request by sgrishchenko**

### CODE REVIEW

File: settings.gradle.kts [hunk 1]
```diff
@@ -1,12 +1,20 @@
 rootProject.name = "kotlin-wrappers"
 
 pluginManagement {
+    repositories {
+        mavenLocal()
```
Comment (turansky) [inline settings.gradle.kts:5]: Is it for `karakum` dependencies?

**Review by turansky — CHANGES_REQUESTED** [review]

File: settings.gradle.kts [hunk 2]
```diff
@@ -1,12 +1,20 @@
 rootProject.name = "kotlin-wrappers"
 
 pluginManagement {
+    repositories {
```
Comment (turansky) [inline settings.gradle.kts:4]: Looks like local development settings

**Review by turansky — CHANGES_REQUESTED** [review]

File: settings.gradle.kts (hunk 2 above)
Comment (sgrishchenko) [reply settings.gradle.kts:4]: I think it is ok, because it is just a prioritized list. I don't know why it is not a default behavior though.

Comment (sgrishchenko) [reply settings.gradle.kts]: It was removed after discussion

**Pull request merged**